vwpy/isnobal_runner.py

```diff
@@ -0,0 +1,6 @@
+from isnobal import isnobal
+import netCDF4
```

Comment:
Please format function calls and signatures with multiple arguments according to PEP8, which dictates spaces after arguments. So instead of

```python
def run_isnobal(input_path=None,output_path=None,event_emitter=None,**kwargs):
```

we want 

```python
def run_isnobal(input_path=None, output_path=None, event_emitter=None, **kwargs):
```

and also not

```python
nc_out = isnobal(input_nc, output_path,event_emitter=event_emitter,**kwargs)
```

but instead

```python
nc_out = isnobal(input_nc, output_path, event_emitter=event_emitter, **kwargs)
```